MAUI/DataForm/getting-started.md

```diff
@@ -7,53 +7,37 @@ control: SfDataForm
 documentation: ug
 ---
 
-# Getting Started with .NET MAUI DataForm (SfDataForm)
+# Getting Started with the .NET MAUI DataForm
 
 This section provides a quick overview of how to get started with the [.NET MAUI DataForm(SfDataForm)](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.html) for .NET MAUI and a walk-through to configure the .NET MAUI DataForm control in a real-time scenario.
```

Comment:
add this line - Follow the steps below to add a basic scheduler view to your project.

Reply:
Added the changes